azure-iot-hub-devicesdk/azure/iot/hub/devicesdk/message.py

```diff
@@ -0,0 +1,53 @@
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
>Copyright [](start = 2, length = 9)

silly request: can you please squash before submitting PRs?  Ideally 1 commit per PR, or 2 if there's different features that are cleanly separated.  In codeflow, I see all 8 iterations at the top, and this one is simple so I only have to look at #8, but it gets more complicated when you add more iterations.  If your updated code (after feedback) ends up is at iteration 12 and your first is at iteration 8, I need to know this to see what you changed.  This can be a pain to figure out.  It's much easier if I can compare iteration 2 to iteration 1 to see what changed.

Reply:
Sure. I generally squash before I am about to merge to master. I assume you speaking from the perspective of code flow. Yes, Its a bit cumbersome to see 8 iterations for 8 commits. From the PR on Github however it looks fine and only shows changes that are made.  

---
In reply to: [237660103](https://github.com/Azure/azure-iot-sdk-python-preview/pull/27#discussion_r237660103) [](ancestors = 237660103)